components/spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import { cn } from '~/libs/utils';
+
+const Spinner = ({ className }: React.HTMLAttributes<HTMLDivElement>) => {
```

Comment:
svg 태그인데 div 엘리먼트에 대한 props를 추가해주신 것 같아요. 적절한 타이핑이 필요할 것 같습니다!

```suggestion
const Spinner = ({ className, ...props }: ComponentProps<'svg'>) => {
```

Reply:
직접 타입을 설정하다 보니 허술하네요... 알려주셔서 감사합니다 
`React.SVGProps<SVGSVGElement>` 로 수정했습니다!

Reply:
확인해보니 `React.SVGProps<SVGSVGElement>`는 `svg` 요소의 props를 추출하는 것이고, 제가 제안 드린 `React.ComponentProps<'svg'>`는 컴포넌트로서의 `svg` 요소의 props를 추출한다고 하네요. 사실 뉘앙스 차이이고, 무얼 사용하던 문제가 발생하는 것은 아닐 것 같지만 의미론적으로 더 정확한 타입을 사용하기 위해 `ComponentProps<'svg>`를 사용하는 것은 어떨지 다시 한 번 여쭤봅니다!ㅎㅎ 근데 사실 문제가 발생하는 것은 아니니 무시하셔도 괜찮습니다!

추가로 나머지 props들도 모두 바인딩 해주시면 좋을 것 같아요!

Reply:
ComponentProps<'svg>로 수정하였습니다! 나머지 props를 바인딩 하라는게 어떤것을 말씀하시는지 여쭤봐도 될까요!